compiler/sem/pragmas.nim

```diff
@@ -296,11 +296,11 @@ proc getOptionalStrLit(c: PContext, n: PNode, defaultStr: string): PNode =
   if n.kind in nkPragmaCallKinds: result = getStrLitNode(c, n)
   else: result = newStrNode(defaultStr, n.info)
 
-proc processCodegenDecl(c: PContext, n: PNode, sym: PSym): PNode =
+proc processCodegenDecl(c: PContext, n: PNode, sym: PSym) =
```

Comment:
The lack of return value sorta breaks the "symmetry" with its sibling routines, but it seemed goofy to maintain it for that reason alone.